src/aleph/vm/orchestrator/resources.py

```diff
@@ -92,6 +93,7 @@ def get_machine_properties() -> MachineProperties:
     )
 
 
+@cors_allow_all
```

Comment:
I'm not sure what the use of this decorator, not just here but in general, since a global cors config was added in supervisor.py?

Reply:
It's a custom decorator that handles the CORS using this library: https://github.com/aio-libs/aiohttp-cors

Reply:
Yeah I have read the decorator code, but you realise it add the same cors config that you also added in supervisor.py in your previous PR? Doesn't that seems a bit strange to you?